bin/out

```diff
@@ -1,5 +1,60 @@
-#!/bin/sh
+#!/usr/bin/env bash
+
```

Comment:
```suggestion
# Does not create an AMI, but generates a Concourse resource version to reflect an already-created AMI. Helpful when a version must be used on-demand rather than waiting for a `check`.
```